### ISSUE TITLE
(FF3PR) Basic Speed/No Encounter/Etc. Functions not Working 

### ISSUE THREAD

**Issue by MinishCrap** [Albeoris/Memoria.FFPR#42]
Hello, my apologies if I'm doing something wrong by posting this as I'm new to GitHub, but I'm having an issue with Final Fantasy III PR using this mod. It worked for Final Fantasy II PR with no issue - I was able to speed up, toggle encounters, and look at hidden items just fine. But for FF III PR, despite installing it as the instructions stated along with the FF3 archive (and removing and redoing it several times) it is just not recognizing these functions. It even takes a minute after first installing the mod for the game to launch, which makes me think it's working, but then nothing changes. I have attached the log file. Does anything seem out of the ordinary? Thank you!

[LogOutput.log](https://github.com/Albeoris/Memoria.FFPR/files/9210386/LogOutput.log)

**Comment by Albeoris** [comment]
@MinishCrap , hi! 

It looks like you forgot to install the mod itself. :)
You unpacked only BeepInEx. Now unpack Memoria.FF3.zip to the game folder.

**Comment by MinishCrap** [comment]
Hi @Albeoris, thanks for the response! After you told me this, I tried once more and it still wasn't working, so I uninstalled and reinstalled FF3 completely and then put in a fresh install of both BepInEx and a the FF3 archive, and _only then_ did my antivirus tell me that it removed the memoria file due to its risk. So I restored it and excluded it from scanning processes, and the mod is working great now. Thanks so much!
